src/lib/viewers/doc/_docBase.scss

```diff
@@ -1,5 +1,6 @@
 @import '~box-ui-elements/es/styles/variables';
 @import '../../boxuiVariables';
+@import '../controls/styles';
```

Comment:
Let's try to avoid this. The controls layer is meant to be isolated/encapsulated from the viewers as much as possible.